packages/fluentui/docs/package.json

```diff
@@ -67,11 +67,13 @@
   },
   "scripts": {
     "build": "gulp build:docs",
-    "build:storybook": "build-storybook -o dist/storybook",
+    "prebuild:storybook": "lage build:info --to @fluentui/react-northstar @fluentui/react-component-ref @fluentui/react-bindings",
+    "build:storybook": "build-storybook --no-manager-cache -o dist/storybook",
     "lint": "eslint --ext .js,.ts,.tsx .",
     "lint:fix": "yarn lint --fix",
     "start": "gulp docs",
     "start:profile": "cross-env NODE_ENV=production PERF=true gulp docs",
+    "prestart:storybook": "lage build:info --to @fluentui/react-northstar @fluentui/react-component-ref @fluentui/react-bindings",
```

Comment:
Looks like a duplicate of the `prebuild:storybook` script above, is this intentional?

Reply:
it's intentional, we wanna provide same DX on local right? without it you'd need to manually run those lage commands, which nobody knows about